astro/src/content/docs/_shared/default-smtp-configuration-properties.mdx

```diff
@@ -0,0 +1,6 @@
+The following property has a default value:
+  * `mail.smtp.ssl.protocols` has a default value of `TLSv1 TLSv1.1 TLSv1.2`.
+    
+Since version `1.44.0`, the following two properties have default values:
+  * `mail.smtp.timeout` has a default value of `2000`.
+  * `mail.smtp.connectiontimeout` has a default value of `2000`.
```

Comment:
I think we should provide example syntax of how to override these values.  E.g.,
```
mail.smtp.timeout=5000
mail.smtp.connectiontimeout=5000
```

Reply:
We don't typically provide example values, but I suppose this one is a bit weird because it is properties syntax instead of normal JSON. Added